.github/workflows/code-quality.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   analyze:
     name: Analyze
     runs-on: ubuntu-latest
-    if: ${{ github.action_repository == 'cloudfoundry/cli' }}
+    if: ${{ github.repository == 'cloudfoundry/cli' }}
```

Comment:
I'd prefer having GitHub Actions related changes in a separate PR, so we can easily synchronize these changes with other branches.